cmd/entrypoint/runner.go

```diff
@@ -168,12 +210,19 @@ func (rr *realRunner) Run(ctx context.Context, args ...string) error {
 // be used for multiple streams if desired.
 // The behavior of the Reader is the same as io.TeeReader - reads from the pipe
 // will be written to the file.
-func newTeeReader(pipe func() (io.ReadCloser, error), path string) (*namedReader, error) {
+func newTeeReader(pipe func() (io.ReadCloser, error), path string, writer io.WriteCloser) (*namedReader, error) {
```

Comment:
This function docstring needs some updates; some of the commentary now belongs on `getFileWriter`.

Reply:
updated the doc strings